src/js/player.js

```diff
@@ -502,7 +502,8 @@ class Player extends Component {
       'autoplay': this.options_.autoplay,
       'preload': this.options_.preload,
       'loop': this.options_.loop,
-      'muted': this.options_.muted
+      'muted': this.options_.muted,
+      'language': this.options_.language
```

Comment:
It should only return the player if an argument is passed to the function.
https://github.com/videojs/video.js/blob/dfd481037e19c2af546ccbc706b7fb42db1c7125/src/js/player.js#L2407